drivers/uktty/Config.uk

```diff
@@ -1,5 +1,6 @@
 menu "Serial console"
 
+source "$(KCONFIG_DRIV_BASE)/uktty/com/Config.uk"
```

Comment:
Will we adopt the `uktty` folder name to `ukconsole` as soon we merge `com` and `ns16550`?

Reply:
We can do it in this PR (separate commit please). On a similar note, we should probably also move the contents of the `drivers/uktty/com` into `drivers/uktty/ns16550`, as these are essentially the same driver.